### DISCUSSION TITLE
Helm chart for K8S?

### DISCUSSION THREAD

**Discussion by lol768** [lukevella/rallly#518] [Ideas]
It's great to see a Docker Compose for self-hosting.

Can I suggest a Helm Chart for easy deployment to K8S?

**Comment by lukevella** [comment]
That sounds like it would be a great contribution to [rallly-selfhosted](https://github.com/lukevella/rallly-selfhosted). Feel free to open a PR there.

**Comment by glazrtom** [comment]
PR here 😌 https://github.com/lukevella/rallly-selfhosted/pull/43
